dockerfiles/hack/run-integration.py

```diff
@@ -50,13 +50,20 @@
 logging.basicConfig(level=logging.INFO,
                     handlers=HANDLERS)
 
+def _parse_dry_run_flag(dry_run: str) -> Optional[str]:
+    dry_run_options = ['--dry-run', '--no-dry-run']
+    if dry_run is not None and dry_run not in dry_run_options:
+        logging.error('Invalid DRY_RUN option given: "%s". Only the following options are allowed: %s', dry_run, dry_run_options)
```

Comment:
Line too long (how came `flake8` doesn't complain)? Also, we are currently using `f-strings` instead of `%s` in log messages - the performance hit is minimal but the messages become easier to read.

Reply:
Oha. Linter wasnt activated :see_no_evil: 

Great catch! :+1: 
Fixed